### DISCUSSION TITLE
NFT跨出藝術走向商用

### DISCUSSION THREAD

**Discussion by HuanYuuuuuu** [1101-GS4719-C/Python-Community-GS4719-C#444] [Q&A]
NFT，又叫做非同質化代幣（Non-fungible token，簡稱「NFTs」），目前主要是應用在電子藝術品上面，可以證明買家確實擁有與特定數位藝術品、音樂或其他作品相連結的無形標記。有很多有錢的投資人會花大筆的金錢購買這些虛擬藝術品。但與其說是購買這些藝術品，比較像是購買這些藝術品的專利。 但其實我對他們放上的照片的所有權的部分其實感覺蠻模糊的
像我想說我也可以把一些東西放上NFT的平台上，於是我把女生服飾店裡給男友做的椅子的照片都放上Open sea (NFT的平台之一)。
但我其實不知道這樣做會不會涉及那些椅子設計師的權利問題等等
之後也想把自己無聊時間畫的畫上傳到NFT平台上，但有些是看著別人的畫風跟著學習的，不知道會不會有抄襲、二創、模仿等等之類的問題
![未命名](https://user-images.githubusercontent.com/99785986/157386836-b0b61936-511d-485a-829c-612c5ec9c20c.png)

**Comment by cordydeng** [comment]
之前有看過一個賣NFT的網站，利用買NFT來兌換實體公仔，可見NFT已經不只能購買虛擬的大頭貼，而是往實體商品的方向邁進。
[新聞點我](https://www.4gamers.com.tw/news/detail/50761/fandora-x-yellow-book-rnft)

**Comment by boson13579** [comment]
作者提到關於NFT, 抄襲、二創、模仿的問題，其實當初NFT藝術一出來的時候，我也思考過同樣的問題，所以我說圖片檔檔案都放公開了，是要買什麼啦...不過這就是一種對藝術家的支持ㄅ.就跟音樂一樣(? youtube 上都可以載,幹嘛買專輯, 而且有了nft之後,更能保證可以支持到真的作者,畢竟有技術保證,所以nft出來,我認為對藝術作者反而是一種保障
